### PR TITLE
Adds SliderLabel Spec and Folder

### PR DESCRIPTION
**Pull request by procload**

### CODE REVIEW

**Review by chrisdholt — APPROVED** [review]

File: change/@fluentui-web-components-085483f0-a46f-4531-b8c4-218a2e78461b.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```
Comment (miroslavstastny) [inline change/@fluentui-web-components-085483f0-a46f-4531-b8c4-218a2e78461b.json:2]:
```suggestion
  "type": "none",
```

As this PR does not affect published package, mark it as "none"

File: packages/web-components/docs/api-report.md [hunk 2]
```diff
@@ -7,16 +7,77 @@
 import { CSSDesignToken } from '@microsoft/fast-foundation';
 import { ElementStyles } from '@microsoft/fast-element';
 import { ElementViewTemplate } from '@microsoft/fast-element';
+import { FASTAccordion } from '@microsoft/fast-foundation';
```
Comment (miroslavstastny) [inline packages/web-components/docs/api-report.md:10]: Could you please rebase and run `yarn buildto web-components` from the repo root? There is no reason spec should contain api.md changes.

Comment (procload) [reply packages/web-components/docs/api-report.md]: Apologies. I'll get this right someday. Updated now.

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/9ae245574502f15ed372dc37b0374883f45ccdf2'>9ae245574502f15ed372dc37b0374883f45ccdf2</a></sub>
<div id="bundleSizeComment9423"/>

**Review by miroslavstastny — APPROVED** [review]

**Pull request merged**